src/EPR.Calculator.API/Program.cs

```diff
@@ -32,6 +33,7 @@
     app.UseSwaggerUI();
 }
 
+app.UseExceptionHandler( _ => { });
```

Comment:
Can you please add a comment and also reply the reference to the URL

Reply:
If we wont add this lambda expression then it throws this exception and gives the below answer to solve.
An error occurred when configuring the exception handler middleware. Either the '_ExceptionHandlingPath_' or the '_ExceptionHandler_' property must be set in 'UseExceptionHandler()'. Alternatively, set one of the aforementioned properties in 'Startup.ConfigureServices' as follows: '**services.AddExceptionHandler(options => { ... });**' or configure to generate a 'ProblemDetails' response in 'service.AddProblemDetails()'.

We can use any one of these to fix.
services.AddExceptionHandler(options => { ... }); -> I used this.

or
combination of these 2 lines
service.AddProblemDetails();
services.AddExceptionHandler();

or app.UseExceptionHandler("/Error") for _ExceptionHandlingPath_ but we don't have any redirect to an error-handling path in web api project.

It’s good to specify either a custom error path ( we don't have now ) or inline error handling logic ( but we we are doing this logic in separate file which is global exception handling file)

Reply:
Please give a URL reference @Mazar-Shaik

Reply:
https://learn.microsoft.com/en-us/aspnet/core/web-api/handle-errors?view=aspnetcore-8.0

Reply:
I can see usage for 
`app.UseExceptionHandler();`
If you agree please change it and remove the blank arrow function @Mazar-Shaik

Reply:
@udaydefra Okay sure I will change but we need to add builder.Services.AddProblemDetails(); as well to work

So 2 lines I'm adding now.
builder.Services.AddProblemDetails()
app.UseExceptionHandler();